nil/tests/journald_forwarder/journald_forwarder_test.go

```diff
@@ -295,6 +295,55 @@ func (s *SuiteJournaldForwarder) TestBatchLogDataInsert() {
 	})
 }
 
+func (s *SuiteJournaldForwarder) TestInsertedValue() {
+	s.Run("Check insert", func() {
```

Comment:
Do you really need to wrap it into `s.Run()`?
It's a single subtest here you can drop that wrapper